web/templates/web/pages/request.html

```diff
@@ -0,0 +1,63 @@
+{% extends "web/pages/base.html" %}
+{% load static %}
+
+{% block styles %}
+  <link rel="stylesheet" href="{% static 'web/css/request.css' %}?" />
+{% endblock styles %}
+
+{% block title %}
+  Запрос от {{ request_owner.username }} - Melomatch
+{% endblock title %}
+
+{% block body %}
+  <section class="hero is-fullheight-with-navbar">
+    <div class="hero-body">
+      <div class="container emoji-container">
+        {% if not request_owner.is_active_link %}
+          <lottie-player src="{% static 'web/images/sad.tgs' %}"
+                         class="is-256x256 mb-5"
+                         autoplay
+                         loop
+                         mode="normal">
+          </lottie-player>
+        {% else %}
+          <figure class="image is-256x256 mb-5">
+            <img class="is-rounded"
+                 height="256"
+                 width="256"
+                 alt="Аватарка"
+                 src="{{ request_owner.avatar }}/islands-200" />
+          </figure>
+        {% endif %}
+        <div class="subtitle text-container">
+          {% if request_owner == request.user %}
+            Попытка сравнить свой музыкальный вкус с самим собой - как пытаться найти отличия с самим собой в зеркале! Давайте найдём того, с кем это делать будет интереснее?
+          {% elif not request_owner.is_active_link %}
+            К сожалению, этот пользователь не открыл свою ссылку для сравнения.
+          {% else %}
+            Готовы узнать, насколько ваш музыкальный вкус схож с <b>{{ request_owner.username }}</b>? Давайте выясним!
+          {% endif %}
+        </div>
+        {% if not request_owner.is_active_link %}
+          <div class="text-container">
+            На данный момент вы не можете сравнить с ним свой музкальный вкус.
+          </div>
+        {% else %}
+          {% if request_owner == request.user %}
+            {# TODO: change link to users search #}
+            <a class="button is-link" href="{% url 'landing' %}">Найти пользователя</a>
+          {% elif request.user.is_authenticated %}
+            {# TODO: change link to compare taste #}
+            <a class="button is-link" href="{% url 'landing' %}">Сравнить вкус</a>
+          {% else %}
+            {% include "web/includes/yandex_button.html" with additional_classes=" is-medium" %}
+          {% endif %}
+        {% endif %}
```

Comment:
Как тебе идея вынести проверку пользователя на один уровень условия вверх? Чтобы кнопка Найти пользователя была доступна даже если у меня будет неактивна эта кнопка
```suggestion
{% if request_owner == request.user %}
            {# TODO: change link to users search #}
            <a class="button is-link" href="{% url 'landing' %}">Найти пользователя</a>
        {% elif not request_owner.is_active_link %}
          <div class="text-container">
            На данный момент вы не можете сравнить с ним свой музкальный вкус.
          </div>
        {% else %}
          {% if request.user.is_authenticated %}
            {# TODO: change link to compare taste #}
            <a class="button is-link" href="{% url 'landing' %}">Сравнить вкус</a>
          {% else %}
            {% include "web/includes/yandex_button.html" with additional_classes=" is-medium" %}
          {% endif %}
        {% endif %}
```

Reply:
Да, я жёстко накосячил с условиями и не перепроверил) Всё подразумевалось, конечно, как ты написала